### DISCUSSION TITLE
Ability to more easily find Custom CSS assigned to template/page, etc.

### DISCUSSION THREAD

**Discussion by AnadarProSvcs** [elementor/elementor#24578] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

In my experience, one of the most consistent frustrations in Elementor is that I know that there is Custom CSS somewhere on a page or template, but I can't figure out where it is.

### Describe the solution you'd like.

It would be great to have an interface that allows for searching of Custom CSS and/or listing of all Custom CSS on a page/template, etc and then allows me to either edit it or easily go to it.  Even just finding it would be a benefit, though.  I think a bit about an IDE. For example, in my IDE, I can search all files for a function/variable, etc and it will list the instances. From there, I can single click and edit directly in the search field or double click and go to the actual file and location. While this would be great, even a search feature with the results would be wonderful.

### Describe alternatives you've considered.

The only practical alternative that I have found is to individually click on elements looking for the Custom CSS.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by jazir555** [comment]
It would also be very useful for the element/section/column etc to have an icon on it when moused over, as well as in the navigator to indicate custom CSS styling. Even better if the icon is clickable and automatically opens to the custom css editor for quick editing.

**Comment by hitsgee** [comment]
This would be amazing. Even if there was a small icon in the navigator to indicate this.
